easycord/plugins/tickets.py

```
@@ -0,0 +1,246 @@
+            return {"category_id": None, "support_role_id": None, "ticket_count": 0}
+        with open(path, encoding="utf-8") as f:
+            return json.load(f)
+
+    def _write_config(self, guild_id: int, config: dict) -> None:
+        path = self._cfg_path(guild_id)
+        tmp = path.with_suffix(".tmp")
+        with open(tmp, "w", encoding="utf-8") as f:
+            json.dump(config, f, indent=2)
+        os.replace(tmp, path)
+
+    def _update_config(self, guild_id: int, **kwargs) -> None:
```

Ruff (0.15.10)

[warning] Missing type annotation for `**kwargs`

(ANN003)